src/libraries/Microsoft.PowerFx.Core/Functions/TexlFunction.cs

```diff
@@ -274,7 +274,12 @@ public virtual async Task<ConnectorSuggestions> GetConnectorSuggestionsAsync(For
         public string LocaleInvariantName { get; }
 
         // A description associated with this function.
-        public string Description => _description(null);
+        public string Description => _description(null);
+
+        /// <summary>
+        /// This function requires an AI disclaimer. 
+        /// </summary>
+        public virtual bool ShowAIDisclaimer => this.Name.StartsWith("AI", StringComparison.OrdinalIgnoreCase);
```

Comment:
if i write a UDF that starts with "AI", would this be true for that? i assume udfs would be created during analysis phase. Do they inherit from this TexlFunction? Can this be overwritten in derived classes?

Reply:
or what if we add a function in future that starts with AI or one of the third party connectors function is prefixed with AI? If this would be overwritten in derived classes then it is fine but startswith(AI) feels like an inefficient approach

Reply:
This is just starting with a whitelist. But I can spell them out until we get the flag propagated from swagger.